pages/recording/[id]/sourcemap/[sourceId].tsx

```diff
@@ -1,5 +1,6 @@
 import { useRouter } from "next/router";
 import React, { useEffect, useState } from "react";
+// eslint-disable-next-line no-restricted-imports
 import { initSocket, client } from "protocol/socket";
```

Comment:
This use is still okay?

Reply:
The sourcemap visualizer is its own beast.